customer-service/omnichannel/smart-assist-bot/README.md

```diff
@@ -1,14 +1,158 @@
+
+    Screenshots for your reference
+
+    ![](screenshots/CDSApp1.png)
+
+    ![](screenshots/CDSApp2.png)
+
+    c. Copy the Application (client) ID and client secret<br />
+
+    ![](screenshots/CDSAppID.png)
+
+    d. Copy the client secret<br />
+
+    ![](screenshots/CDSSecret.png)
+
+    Add API permissions to this app inorder to access CDS capabilities
+
+    ![](screenshots/APIPermissions1.png)
+
+    ![](screenshots/APIPermissions2.png)
+
+    ![](screenshots/APIPermissions3.png)
+
+
+    d. Update the copied App Id and secret to appsettings.json<br />
+
+     ```//Dynamics connection settings
+    "DynamicsAppId": "<Azure Application Id copied in step 3c, This enables Bot to talk to CDS>",
+    "DynamicsAppSecret": "<App secret for CDS App Id copied in step 3d>",
+    "DynamicsOrgUrl": "<CDS Org Url>",
+    "TenantId": "<CDS Tenant Id>"```
+
+  Please use the same app ID while [creating a bot user](https://docs.microsoft.com/en-us/dynamics365/omnichannel/administrator/smart-assist#step-2-create-a-bot-user) later. Also if your bot needs to search for knowledge base articles, then you need to provide either
+  the **Customer Service Manager** or **Customer Service Representative** role to your bot user.
+
+**Publish your WebApp to azure**<br />
+
+- Build your code and publish from visual studio like mentioned in this [link](https://docs.microsoft.com/en-us/visualstudio/deployment/quickstart-deploy-to-azure?view=vs-2019)
+- Copy the WebApp's endpoint which opens up in a browser after publishing. For Instance, lets say webapp endpoint is https://smartassistbot2020.azurewebsites.net/
+- Or you can also copy the endpoint from azure portal.<br />
+Go to your webapp and select **Overview** from left pane (see the screenshot below).
+
+![](screenshots/WebappsEndpoint.png)
+
+
+**Update Messaging endpoint of Bot channels registration**<br />
+
+- Now update the Messaging endpoint of Bot channels registration, created in 2(a) with the WebApps messaging endpoint created in previous step
+**Messaging endpoint will be ```WebApp Endpoint + /api/messages```**  (i.e)  
+_In our Example, it will be : https://smartassistbot2020.azurewebsites.net/api/messages_
+
+![](screenshots/endpointupdate.png)
+
+**Enable Teams channel**
+
+Please make sure to enable Teams channel like mentioned in this [link](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0).
+
+![](screenshots/channel1.png)
+
+![](screenshots/Teams2.png)
+
+**Test your Smart bot**
+
+- Test your bot online using ```Test in Web Chat``` option. See [here](https://docs.microsoft.com/en-us/azure/bot-service/abs-quickstart?view=azure-bot-service-4.0#test-the-bot) for more details. 
+- If you are using trail org use keywords `travel`  or `book` or `audio equipment` to see the smart suggestions
+
+### 2 . Integrating Smart Assist to an exisiting Azure Bot
```

Comment:
Consult whether it makes sense to have a section 2

Reply:
Will check with Neerja and close this in next PR